test/test_populationstrategy.py

```diff
@@ -108,3 +109,7 @@ def test_transitions_not_modified(population_strategy: PopulationStrategy):
                " modified the transitions".format(population_strategy))
 
     assert same, err_msg
+
+   
+def test_ListPopulationSize():
+    pop_size = pyabc.ListPopulationSize(values=[10, 5, 1.5])
```

Comment:
:) the population size will be rewritten soon, so fine not to have real tests for now.